### PR TITLE
Generate Decay Matrix

### PR DESCRIPTION
**Pull request by scopatz**
This builds off of #1 and adds the ability to include a decay matrix to the generated code.

### CODE REVIEW

**Comment by scopatz** [comment]
Also, this is needed for the Cyclus integration.

File: transmutagen/gensolve.py [hunk 1]
```diff
@@ -5,6 +5,15 @@
...
 from sympy import im
+import numpy as np
+
+try:
+    import pyne.utils
+    import pyne.data
+    from pyne import nucname
+    HAVE_PYNE = True
+except ImportError:
+    HAVE_PYNE = False
```
Comment (asmeurer) [inline transmutagen/gensolve.py]: I believe pyne is already a dependency.

File: transmutagen/gensolve.py [hunk 2]
```diff
@@ -217,14 +254,50 @@ def make_ijk(ij, N):
...
 
+
+def pyne_decay_matrix(fromto):
+    if not HAVE_PYNE:
+        raise RuntimeError('Must have PyNE installed to generate decay matrix from '
+                           'PyNE data. Please either install PyNE or generate with '
+                           'decay_matrix_kind="none", or --decay-matrix=none from the '
+                           'command line.')
+    if pyne.utils.use_warnings():
+        pyne.utils.toggle_warnings()
```
Comment (asmeurer) [inline transmutagen/gensolve.py]: Does this disable them if they are enabled? I think we might already call this elsewhere.

Comment (scopatz) [reply transmutagen/gensolve.py]: We do, but it wasn't working. I think it might be called twice elsewhere.

File: transmutagen/gensolve.py (hunk 1 above)
Comment (scopatz) [reply transmutagen/gensolve.py]: Yeah, you are right. I didn't know if it was a hard dep when I first started this branch.

**Comment by scopatz** [comment]
OK looks like this is fixed now.

**Comment by asmeurer** [comment]
I don't know if it's ready yet but if it is feel free to merge (what's there looks good).

File: transmutagen/gensolve.py (hunk 2 above)
Comment (asmeurer) [reply transmutagen/gensolve.py]: OK. I really don't understand how this pyne warning stuff works. If we should refactor it somehow let me know.

Comment (scopatz) [reply transmutagen/gensolve.py]: Will do!

**Comment by scopatz** [comment]
OK Thanks!

**Pull request merged**